webviews/src/open-chat/styles.css

```
@@ -0,0 +1,555 @@
+  --agent-control-hover: color-mix(in srgb, var(--cmux-diff-bg) 68%, var(--cmux-diff-fg) 32%);
+  --agent-border: color-mix(in srgb, var(--cmux-diff-fg) 12%, transparent);
+  --agent-border-strong: color-mix(in srgb, var(--cmux-diff-fg) 20%, transparent);
+  --agent-text: var(--cmux-diff-fg);
+  --agent-muted: color-mix(in srgb, var(--cmux-diff-fg) 56%, var(--cmux-diff-bg));
+  --agent-soft: color-mix(in srgb, var(--cmux-diff-fg) 74%, var(--cmux-diff-bg));
+  --agent-accent: var(--cmux-diff-accent);
+  --agent-accent-soft: color-mix(in srgb, var(--cmux-diff-accent) 20%, transparent);
+  --agent-danger: #ff6b57;
+  --agent-warning: var(--color-token-charts-orange, #fb6a22);
+  --agent-shadow: 0 18px 55px color-mix(in srgb, #000 34%, transparent);
+  background: var(--agent-page-bg);
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)